package.json

```diff
@@ -248,7 +248,7 @@
     "react-intl": "2.7.2",
     "react-lottie": "1.2.3",
     "react-markdown": "4.3.1",
-    "react-polymorph": "1.0.3",
+    "react-polymorph": "1.0.2",
```

Comment:
is there any specific reason for lowering the react-polymorph version? Is it particular to the design aspect of the UI-components like dropdowns in this case.

Reply:
Please take a look at the comments in the ticket.

Reply:
@przemyslaw-wlodek are we sure that this **only** reverts this specific change and not other stuff as well?

Reply:
Yeah. That's my concern as well. Looks like it also contains changes from 1.0.2:
https://github.com/input-output-hk/react-polymorph/pull/199

Reply:
I tested it locally and seems to be fine

Reply:
Yes, The changes in changelog are intentional and discussed with @danielmain and @dmitrii-gaico.
There's nothing else than a revert aparat from that.